config.yaml

```diff
@@ -0,0 +1,4 @@
+options:
+  install_sources:
+    default: |
+      - ppa:openstack-charmers/vaultlocker
```

Comment:
How will this impact CDK in offline environments (as documented [here](https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Running-CDK-in-a-restricted-environment))?

Seems like the operator will have to add this PPA to their apt mirror, and then deploy the charm with `install_sources=''` to prevent apt errors from occurring. Does that sound right? If so, is there any way we can make this friendlier?

Reply:
I think only one package would need to be added to the mirror, and then the config updated.  AFAIK this is pretty standard for charms that use apt packages and is why it's supported directly in the apt layer.  I'm not sure of how we could make it any cleaner.  Maybe add a placeholder resource for the package?  I'm not sure if that would be easier to use or not.

Reply:
Yeah, I can't think of anything better either to be honest. I think this is acceptable, but we have to update the offline documentation to work with this change. Can you please make sure that happens?

Reply:
Sure thing.